portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -37,6 +37,8 @@
   private static final int DEFAULT_MESSAGES = 10;
   private static final int MAX_MESSAGES = 50;
 
+  DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
```

Comment:
Does this work if it's `private final` instead of package-private non-final? (if not don't sweat it)

Reply:
Yep, It's been changed.